ionic2/app/services/api.service.ts

```diff
@@ -284,4 +284,30 @@ export class ApiService {
     });
     return request.map(ApiService.handleResponse(Sighting));
   }
+
+  getTypes(): string[] {
+    const types: string[] = [
```

Comment:
In the API Service, but the method returns the const that has been imported from another file